### PR TITLE
Fix/ledger integration imported label

### PR DESCRIPTION
**Pull request by fstream9**
**Description**

Fixes bug where ledger accounts showing up with label imported.  Now the label will be 'HARDWARE' in both the drawer and accounts list components.

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses #???

### CODE REVIEW

File: app/components/UI/AccountList/AccountElement/index.js [hunk 1]
```diff
@@ -116,24 +119,32 @@ class AccountElement extends PureComponent {
...
-		const imported = isImported ? (
-			<View style={styles.importedWrapper}>
-				<Text numberOfLines={1} style={styles.importedText}>
-					{strings('accounts.imported')}
-				</Text>
-			</View>
-		) : null;
+
+		const keyRingLabel =
+			keyringType !== 'HD Key Tree' ? (
```
Comment (zoltanarvai) [inline app/components/UI/AccountList/AccountElement/index.js:135]: technically they can have multiple keyring types
"simple" "HD Keyring Tree" "QRKeyring" and whatever they add in the future

So I would be quite specific and say if it's Ledger then do this in any other case do something else.

File: app/components/UI/AccountList/AccountElement/index.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ const createStyles = (colors) =>
 			color: colors.error.default,
 			marginLeft: 4,
 		},
-		importedView: {
```
Comment (zoltanarvai) [inline app/components/UI/AccountList/AccountElement/index.js:55]: is there a reason why we changed the names of the styles?

**Review by zoltanarvai — COMMENTED** [review]
So basically we can have:
- Imported (most likely an HD Tree type)
- Hardware (Ledger of QR??)
- No label (native MetaMask account: Simple, HD, what else?)

File: app/components/UI/AccountList/AccountElement/index.js (hunk 1 above)
Comment (fstream9) [reply app/components/UI/AccountList/AccountElement/index.js:135]: That's what I thought would be the case too.  This code builds on top of the previous logic without changing it, only enhances, meaning that it does exactly as it was doing before except now if it sees Ledger will display Hardware label and then if they want to add anything else it's easily extensible.

File: app/components/UI/AccountList/AccountElement/index.js (hunk 2 above)
Comment (fstream9) [reply app/components/UI/AccountList/AccountElement/index.js:55]: these stylings were for the imported label and didn't make sense to keep them specific so made it generic as they're not only for imported label anymore

**Review by zoltanarvai — APPROVED** [review]

File: app/components/UI/AccountList/AccountElement/index.js (hunk 1 above)
Comment (zoltanarvai) [reply app/components/UI/AccountList/AccountElement/index.js:135]: Makes sense

**Pull request merged**